adapter/stdioLogger/stdioLogger.go

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 // Package stdioLogger provides an implementation of the mixer logger aspect
-// that writes logs (serialized as JSON) to a standard expectedStream (stdout | stderr).
+// that writes logs (serialized as JSON) to a standard stream (stdout | stderr).
```

Comment:
My comment relates to the PayloadFormat stuff. What's the different between TEXT and STRUCTURED then?